### PR TITLE
fix: intro to quickstart change

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: docs/intro.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Introduction
+title: Quick Start
```
Comment (utkarsha-deriv) [inline docs/intro.mdx:2]: Label should be Quick start with small 's'

File: docs/intro.mdx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 hide_title: false
 draft: false
-sidebar_label: Introduction
+sidebar_label: Quickstart
```
Comment (utkarsha-deriv) [inline docs/intro.mdx:5]: Same here, label = Quick start

File: docs/intro.mdx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 sidebar_position: 0
 sidebar_class_name: hide-sidebar-item
 tags:
-  - introduction
+  - Quick Start
   - getting started
   - quick startup
```
Comment (utkarsha-deriv) [inline docs/intro.mdx:11]: we can remove quick startup from here

File: docs/intro.mdx [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+title: Quick Start
 hide_title: false
 draft: false
-sidebar_label: Introduction
+sidebar_label: Quickstart
 sidebar_position: 0
 sidebar_class_name: hide-sidebar-item
 tags:
-  - introduction
+  - Quick Start
```
Comment (utkarsha-deriv) [inline docs/intro.mdx:9]: small 's' here too. thanks

**Review by utkarsha-deriv — CHANGES_REQUESTED** [review]

**Review by hubert-deriv — APPROVED** [review]

**Pull request merged**